app/services/staff_nomis_checker_factory.rb

```diff
@@ -0,0 +1,10 @@
+class StaffNomisCheckerFactory
+  def self.for(visit)
+    case visit.processing_state
+    when 'requested'
```

Comment:
doesn't matter much but it's a bit more elegant and convenient to use constants for these types of things.